.github/workflows/tests.yml

```diff
@@ -87,6 +87,11 @@ jobs:
         source '/usr/share/miniconda/etc/profile.d/conda.sh'
         conda activate improver_${{ matrix.env }}
         black --check .
+    - name: flake8
+      run: |
+        source '/usr/share/miniconda/etc/profile.d/conda.sh'
+        conda activate improver_${{ matrix.env }}
+        flake8 improver improver_tests
     - name: pylint
```

Comment:
I think we can remove pylint now

Reply:
Could we also replace the pylint command in bin/improver-tests in favour of flake8?

Reply:
Good idea. I'll do this.

There's a few files which have `# pylint: disable=...` annotations. Perhaps you could remove these in #1466?